src/planscape/users/forms.py

```diff
@@ -0,0 +1,46 @@
+from allauth.account.adapter import get_adapter
+from allauth.account.forms import default_token_generator
+from allauth.account import app_settings
+from allauth.account.utils import user_pk_to_url_str
+
+from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+
+from dj_rest_auth.forms import AllAuthPasswordResetForm
+
+
```

Comment:
Please provide a class level comment describing the purpose of this class for future maintainers